teammapper-frontend/src/app/modules/about/components/jumbotron/jumbotron.component.scss

```diff
@@ -2,7 +2,7 @@
 @import 'src/theme.scss';
 
 section.jumbotron {
-  background-color: mat.get-color-from-palette($app-primary);
+  background-color: mat.m2-get-color-from-palette($app-primary);
```

Comment:
why do we have to use the older version here? Guess the default is v3 now? Was the design broken? We can stay with version 2 but then we should create a new issue about updating to material v3. But lets only consider this if there were problems.

Reply:
This was done by the auto-updater, because `mat.get-color-from-palette` is not a valid function (only if prefixed by `m2-`, same goes for variables).

Reply:
I created a new issue for this #367